big_tests/test.config

```diff
@@ -227,20 +227,21 @@
         {auth_method, <<"SASL-ANON">>}]}
                      ]}.
 
+%% All options "scope" which have value "host" should be tested with option "global" locally before creating PR.
```

Comment:
I think we could simply test it with `scope = global` in some of the presets below, that's what we discussed at the daily.